pom.xml

```diff
@@ -41,9 +41,11 @@
     </scm>
 
     <properties>
+        <gpg.skip>true</gpg.skip><!-- by default skip gpg -->
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.outputTimestamp>2022-02-30T14:33:00Z</project.build.outputTimestamp>
-        <gpg.skip>true</gpg.skip><!-- by default skip gpg -->
```

Comment:
Why did you remove this?

Reply:
I did not, it is two lines above.

Reply:
In main is in that order, that is why I moved it up.

Reply:
OK. Looks good then.